tools/data-populate-cloud.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+#
+# Populates a Terra Data Repo instance with a dataset, test data, and a snapshot.
```

Comment:
How is this different from the existing code in `tools/setupResourceScripts`?

That code already supports creating a dataset, ingesting data, and creating snapshots

Reply:
We chatted a bit about this in the TDR sync yesterday, but in short I've had a number of issues trying to get the `setupResourceScripts` running, including:
* Issues with `gcloud auth` when using a `venv`
* Issues with SSL/HTTPS
* Issues with missing files or permissions
* Issues with debugging Python errors

The `setupResourceScripts` also seem very geared towards the Data Exploration use case, but less generally for quick iteration and testing.

As a result it was faster for me to iterate on a script to create a simple dataset and snapshot.

I also mainly posted this to help my team members that have also had issues onboarding to TDR, and not necessarily as a real PR to merge. From my perspective, there are two problems relating to setting up TDR that I think we should improve:
1. Getting into a state where you can actually `bootRun` the code. This requires valid credentials, appropriate environment variables, a running database, and a built codebase with all the generated code. This is what I aim to solve with the `bootrun.sh` script.
2. Creating a valid dataset and snapshot for testing iteration. This requires a billing profile, dataset with some simple structure, ingesting data, and making some sort of snapshot of that dataset. This is what I aim to solve with the `data-populate-cloud.sh` script (which itself is really a derivative of `data-populate.sh` script).

I don't have a strong opinion on whether either of these should be in the `setupResourceScripts` or in another part of the code, but the current state did not address our pressing need to be up and running. What I would ultimately want is something as simple as:

```
make build
make run
make test
```

And everything would work out of the box.

Reply:
OK, if you do want to make a real PR for this then maybe the best way forward is to split out the ingest changes from the bootRun changes since the bootRun scripts don't require the ingest script. I think it'd be great to have scripting around getting bootRun working and I wouldn't want to hold up those changes on a discussion of the ingest side of things.

I do think that we'd be better off having a single script for ingest, and that we'd be better off improving / fixing the python code rather than starting anew in bash.